common/quotas/global/collection/collection_test.go

```diff
@@ -38,6 +38,7 @@ import (
 
 	"github.com/uber/cadence/common/clock"
 	"github.com/uber/cadence/common/dynamicconfig"
+	dynamicQuotas "github.com/uber/cadence/common/dynamicconfig/quotas"
```

Comment:
Still not sure we should have this hierarchy, but i think package name are usually all lowercase 
```suggestion
	dynamicquotas "github.com/uber/cadence/common/dynamicconfig/quotas"
```